jmbitcoin/jmbitcoin/secp256k1_transaction.py

```diff
@@ -98,18 +111,18 @@ def read_flag_byte(val):
     for i in range(ins):
         obj["ins"].append({
             "outpoint": {
-                "hash": read_bytes(32)[::-1],
+                "hash": hex_string(ser_read(tx, 32), hexout),
                 "index": read_as_int(4)
             },
             #TODO this will probably crap out on null for segwit
-            "script": read_var_string(),
+            "script": hex_string(read_var_string(), hexout),
```

Comment:
So seems I'm getting `SerializationTruncationError` when trying to read some scripts, are they supposed to actually exist here in the tests that are failing?